integration/e2e/utils.go

```diff
@@ -0,0 +1,85 @@
+package e2e
+
+import (
+	"github.com/hyperledger/fabric/integration/nwo"
+)
+
+type ChaincodeQueryWithHex struct {
+	ChannelID string
+	Name      string
+	Ctor      string
+}
+
+func (c ChaincodeQueryWithHex) SessionName() string {
+	return "peer-chaincode-query"
+}
+
+func (c ChaincodeQueryWithHex) Args() []string {
+	return []string{
+		"chaincode", "query",
+		"--channelID", c.ChannelID,
+		"--name", c.Name,
+		"--ctor", c.Ctor,
+		"--hex",
+	}
+}
+
+func Build() *nwo.Components {
+	components := &nwo.Components{}
+	components.Build()
+	return components
+}
+
```

Comment:
I added this function because we needed to build fab3. Since you don't, you could directly just call components.Build(). You don't need this extra helper function.

Reply:
removed it